policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_eligibility(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec family allowance eligible indivudual"
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_8"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.fa
+
+        person = household.members
+        # is parent and resides with of the eligible dependent child
```

Comment:
No age requirements or anything?

Reply:
No, “eligible individual”, in respect of an eligible dependent child, at any time means an individual who at that time
(a)  resides with the eligible dependent child;
(b)  is the father or mother of the eligible dependent child;
https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_8

Age requirement applies only to dependent children, checked in `qc_fa_payment`